### PR TITLE
Validations to Pass CSI Sanity Tests and Script

### PR DESCRIPTION
**Pull request by washcycle**
-added validations to pass csi-sanity testing
-removed deprecated/unneeded  passthrough import
-added test.sh (needs work)
-needs CI/CD still for github actions

### CODE REVIEW

File: _output/seaweedfs-csi-driver
Comment (chrislusf) [inline _output/seaweedfs-csi-driver:1]: Remove

File: cmd/seaweedfs-csi-driver/__debug_bin
Comment (chrislusf) [inline cmd/seaweedfs-csi-driver/__debug_bin:1]: Remove

File: seaweedfs-csi-driver
Comment (chrislusf) [inline seaweedfs-csi-driver:1]: Remove

**Comment by washcycle** [comment]
Oops didn't mean to add those I'll fix that.

**Comment by washcycle** [comment]
-removed the binaries
-also noticed I had some pkgs I wasn't using for csi-test, I went with the download option vs the unit test. I'm not familiar enough with go unit test to make it from scratch yet.

I got the results down to this

```
Summarizing 7 Failures:
  [FAIL] Controller Service [Controller Server] ControllerPublishVolume [It] should fail when the volume does not exist
  /root/development.local/csi-test/pkg/sanity/controller.go:959
  [FAIL] Controller Service [Controller Server] ControllerPublishVolume [It] should fail when the node does not exist
  /root/development.local/csi-test/pkg/sanity/controller.go:998
  [FAIL] Controller Service [Controller Server] volume lifecycle [It] should be idempotent
  /root/development.local/csi-test/pkg/sanity/controller.go:1684
  [FAIL] Node Service NodeGetInfo [It] should return appropriate values
  /root/development.local/csi-test/pkg/sanity/node.go:369
  [FAIL] Node Service NodeUnpublishVolume [It] should remove target path
  /root/development.local/csi-test/pkg/sanity/node.go:478
  [FAIL] Node Service NodeExpandVolume [It] should fail when volume is not found
  /root/development.local/csi-test/pkg/sanity/node.go:782
  [FAIL] Node Service [It] should be idempotent
  /root/development.local/csi-test/pkg/sanity/node.go:79

Ran 36 of 78 Specs in 0.282 seconds
FAIL! -- 29 Passed | 7 Failed | 1 Pending | 41 Skipped
```

**Comment by washcycle** [comment]
@chrislusf this is ready I think

I tried to find an easy way to pass some of the tests that requiring listing the volumes to see if the new one already exists, but that seems to be extremely difficult to implement without tying into the kube client. 

I just left in my validations and a few other tweaks.

**Pull request merged**